### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by alainlankers** [bcyran/fancy-motd#15]
Hello, adding temperatures would be a nice feature!

**Comment by bcyran** [comment]
Hey @alainlankers! I finally got around to adding the module (https://github.com/bcyran/fancy-motd/commit/09dd6e574f1d5514a5223827d9965b8fc26a3813), it depends on `lm-sensors`. Check it out!

![2022-08-07_13-26](https://user-images.githubusercontent.com/8322846/183288693-2020887e-904e-4229-b05a-b2d94a8e2638.png)

**Comment by alainlankers** [comment]
Thanks so much for the update! The module is showing up fine but it gives me no temperature. But I think this is due to my sensors. Not because of the module.

**Comment by bcyran** [comment]
What's the output of `sensors`? I believe you have to run `sensors-detect` after installing `lm-sensors` for the first time.

**Comment by alainlankers** [comment]
This is the output of sensors:
iwlwifi_1-virtual-0
Adapter: Virtual device
temp1:        +38.0°C  

amdgpu-pci-2f00
Adapter: PCI adapter
vddgfx:        6.00 mV 
fan1:           0 RPM  (min =    0 RPM, max = 3300 RPM)
edge:         +36.0°C  (crit = +110.0°C, hyst = -273.1°C)
                       (emerg = +115.0°C)
junction:     +36.0°C  (crit = +105.0°C, hyst = -273.1°C)
                       (emerg = +110.0°C)
mem:           +0.0°C  (crit = +105.0°C, hyst = -273.1°C)
                       (emerg = +110.0°C)
power1:        4.00 W  (cap = 135.00 W)

k10temp-pci-00c3
Adapter: PCI adapter
Tctl:         +44.9°C  
Tdie:         +44.9°C  
Tccd1:        +34.5°C

**Comment by TCB13** [comment]
> Thanks so much for the update! The module is showing up fine but it gives me no temperature. But I think this is due to my sensors. Not because of the module.

I'm having the same problem in an AMD Ryzen 5 1600 system.

````
Uptime:        1 week, 19 hours, 38 minutes
Load average:  0.01, 0.06, 0.07
Temperatures:  °C
````
````
$ sensors
k10temp-pci-00c3
Adapter: PCI adapter
Tctl:         +31.2°C
Tdie:         +31.2°C
````

**Comment by bcyran** [comment]
Should be fixed by e3d3dff71630c38fb117ddac05155d7984d8d4c9
